ui/lib/pki/addon/routes/overview.js

```diff
@@ -11,34 +11,50 @@ export default class PkiOverviewRoute extends Route {
     return this.window || window;
   }
 
-  hasConfig() {
+  async hasConfig() {
     // When the engine is configured, it creates a default issuer.
     // If the issuers list is empty, we know it hasn't been configured
     const endpoint = `${this.win.origin}/v1/${this.secretMountPath.currentPath}/issuers?list=true`;
-    return this.auth
+
+    return await this.auth
```

Comment:
We don't need the async or await since this is already returning the promise

Reply:
Thanks for catching that!